.travis.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       node_js: "8" # This *has* to be the "build leader"
       script: skip
       after_success:
-        - TRAVIS_JOB_NUMBER=WORKAROUND.1 npm run semantic-release
```

Comment:
no other libs use this, is this needed any more?

Reply:
Not needed, this predates the use of travis build stages.

Reply:
See here: https://github.com/snyk/snyk-php-plugin/pull/23